src/jobs/enrich-mustache-from-path-filter.yml

```diff
@@ -0,0 +1,53 @@
+  # meant to be a drop in replacement for path-filtering/filter
```

Comment:
I think this is good to set as an actual description instead of just yaml comments? In the description, also linking to the original path-filtering orb would be helpful.